pkg/splunk/enterprise/afwscheduler.go

```diff
@@ -1355,7 +1355,7 @@ func (shcPlaybookContext *SHCPlaybookContext) runPlaybook(ctx context.Context) e
 	var err error
 	var ok bool
 	cr := shcPlaybookContext.cr.(*enterpriseApi.SearchHeadCluster)
-	if cr.Status.Phase != splcommon.PhaseReady {
+	if cr.Status.Phase != enterpriseApi.PhaseReady {
```

Comment:
are you sure this change is correct? shouldn't this be `splcommon.PhaseReady`?

Reply:
Yes, @akondur has done changed on develop branch to use enterpriseApi , i think the Spec and PhaseReady is moved from splcommon to enterpriseApi